pom.xml

```diff
@@ -238,6 +238,14 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>3.1.11</version>
+                <configuration>
+                    <excludeFilterFile>spotbugs-exclude.xml</excludeFilterFile>
```

Comment:
As discussed in person, I don't think we want to use an xml file for exclusions, I think we want to use annotations and there is one just for this case `edu.umd.cs.findbugs.annotations.SuppressFBWarnings` so I think we should add it to the relevant places instead. :smiley: